### ISSUE TITLE
With a lot of text we need a scroll bar

### ISSUE THREAD

**Issue by Seko1900** [TokelPlatform/tokel_dapp#237]
I use Ubuntu 20.01, but it will probably be the same on other OS. If the token has a lot of text, it cannot be read to the end. We need a scrollbar here in the optimal case.

![Bildschirmfoto vom 2021-12-15 08-59-32](https://user-images.githubusercontent.com/49843358/146147418-ab5d362a-b5ac-4172-861f-3aa0a751b366.png)

**Comment by piggydoughnut** [comment]
@Seko1900 could you please confirm if that is still an issue with the latest release of the dapp? Thank you

**Comment by NutellaLicka** [comment]
Yes this is still an issue. There is no scroll bar for the information viewport
https://gyazo.com/464610e06677296a78c2a10a84e2e427
